.github/workflows/biased-lang.yml

```diff
@@ -0,0 +1,18 @@
+name: Detecting Biased Language
+on:
+  pull_request:
+    branches:
+      - main
+jobs:
+  biased_lang:
+    runs-on: ubuntu-latest
+    name: Detecting Biased Language
+    steps:
+      - uses: actions/checkout@v2
+      - id: pink-panther
+        uses: splunk/pink-panther@main
+        with:
+          token: ${{secrets.GITHUB_TOKEN}}
+          path: $GITHUB_WORKSPACE
```

Comment:
Do we need these three fields? They seem like default values

Reply:
Ah looks like we don't! Thought we needed to pass these through but they look accessible from the action itself. I will update, thanks!